ee-9/feature-pack/src/license/feature-pack-licenses.xml

```diff
@@ -28,6 +28,22 @@
         </license>
       </licenses>
     </dependency>
+    <dependency>
+      <groupId>jakarta.authentication</groupId>
+      <artifactId>jakarta.authentication-api</artifactId>
+      <licenses>
+        <license>
+          <name>Apache License 2.0</name>
```

Comment:
I think this should be changed to Eclipse Public License - v 2.0 as per https://github.com/eclipse-ee4j/authentication/blob/master/LICENSE.md#eclipse-public-license---v-20

The below GPL v2 with classpath exception matches https://github.com/eclipse-ee4j/authentication/blob/master/LICENSE.md#the-gnu-general-public-license-gpl-version-2-june-1991 + https://github.com/eclipse-ee4j/authentication/blob/master/LICENSE.md#classpath-exception

Reply:
@scottmarlow thanks; fixed